CONTRIBUTING.md

```diff
@@ -96,6 +96,13 @@ Plugins tests are run automatically on contributions using Cirrus CI. However, d
 cost constraints, pull requests from non-committers may not run all the tests
 automatically.
 
+The plugins team prefers that unit tests are written using `setMockMethodCallHandler`
```

Comment:
Note that this will require an update on the next stable release as setMockMethodCallHandler is deprecated.